charmhelpers/contrib/openstack/context.py

```diff
@@ -1761,6 +1761,7 @@ def __call__(self):
             "public_processes": int(math.ceil(self.public_process_weight *
                                               total_processes)),
             "threads": 1,
+            "disable_wsgi_rotation": config('disable-wsgi-rotation'),
```

Comment:
IIRC running `config('non-defined-config')` will fail, so introducing this change will break charms that don't have this config option defined.

Reply:
has changed the option disable-wsgi-rotation to wsgi-rotation

Reply:
and for non-defined-config, I also add the following 3 lines

+        enable_wsgi_rotation = config('wsgi-rotation')                         
+        if enable_wsgi_rotation is None:                                       
+            enable_wsgi_rotation = False